pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -342,10 +344,12 @@ func (rc *reconciler) reconcile() {
 			} else {
 				// Volume is attached to node, detach it
 				if rc.controllerAttachDetachEnabled || !attachedVolume.PluginIsAttachable {
-					// Kubelet not responsible for detaching or this volume has a non-attachable volume plugin,
-					// so just remove it to actualStateOfWorld without attach.
-					rc.actualStateOfWorld.MarkVolumeAsDetached(
-						attachedVolume.VolumeName, rc.nodeName)
+					// Kubelet not responsible for detaching or this volume has a non-attachable volume plugin.
```

Comment:
All the comments seem to be after the conditional so I left that as is. Do you mean move it to line 345?